### DISCUSSION TITLE
A couple ideas Party kitty and generic notes section

### DISCUSSION THREAD

**Discussion by MythEvolution** [fmasa/wfrp-master#47] [Ideas]
A party kitty section

And

A generic notes section to add,  title and description.

A place to add on going character information,  plot lines, reputation, NPCs, contacts etc.

Thanks for excellent app.

**Comment by fmasa** [comment]
Hi,
wouldn't it be better to integrate something like [World Anvil](https://www.worldanvil.com/)? This seems very system-agnostic to include in the WFRP app.

**Reply by MythEvolution** [reply]
I did not know about world anvil. I'll check it out. 

Thanks
